src/frontend/Sider/Behandling/Inngangsvilkår/Målgruppe/EndreMålgruppeRad.tsx

```diff
@@ -66,14 +70,21 @@ const EndreMålgruppeRad: React.FC<{
 
             const erNyPeriode = målgruppe === undefined;
 
-            return request<Målgruppe, EndreMålgruppeForm>(
+            return request<LagreVilkårperiodeResponse<Målgruppe>, EndreMålgruppeForm>(
                 erNyPeriode ? `/api/sak/vilkarperiode` : `/api/sak/vilkarperiode/${målgruppe.id}`,
                 'POST',
                 målgruppeForm
             )
                 .then((res) => {
                     if (res.status === RessursStatus.SUKSESS) {
-                        erNyPeriode ? leggTilMålgruppe(res.data) : oppdaterMålgruppe(res.data);
+                        erNyPeriode
+                            ? leggTilMålgruppe(res.data.periode)
+                            : oppdaterMålgruppe(res.data.periode);
+                        if (res.data.stønadsperiodeStatus === StønadsperiodeStatus.Ok) {
+                            settStønadsperiodeFeil(undefined);
```

Comment:
Her kan man potensielt sett overskrive feilmeldinger som allerede ligger i stønadsperiodekomponenten 🤔

Reply:
Og burde vi sette stønadsperioder i redigeringsmodus?

Reply:
Egentligen skal man kun kunne endre vilkårperiode eller stønadsperiode, dvs man skal ikke kunne endre begge samtidig. Så det burde då egentlige ikke kunne finnes en feil fra før 👀

Reply:
Dgb for nå, men kan se på å bare tillate et felt i redigering om gangen for å slippe det problemet